metrics/prometheus.go

```diff
@@ -1079,6 +1079,23 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					}
 				},
 			},
+			{
+				name:        "container_root_process_ulimits_soft",
```

Comment:
you can remove "root_process" from the name.  That is implied with other metrics (e.g. threads_max).  You can leave it in the help text if you want.